composer.json

```diff
@@ -100,6 +100,7 @@
         "drupal/azure_mailer": "^1.0",
         "drupal/color_field": "^3.0",
         "drupal/config_ignore": "^2.4",
+        "drupal/config_ignore_auto": "^2.0",
```

Comment:
- v3 requires config_ignore v3
- we use config_ignore v2 at the moment
- if i try to update to config_ignore v3, it says a patch needs to be removed
- when i look at the patch details: https://www.drupal.org/project/config_ignore/issues/3252291 the latest answer scares me:

```quote
This issue is being closed because Config Ignore 2.x is deprecated.

The new 3,x version has been re-written from the ground up and works in a totally different way, yet it provides the same backwards compatible functionality. All the 2.x tests pass in 3.x and the 3.x codebase is easier to maintain.
Users of 2.x are strongly encouraged to upgrade to 3.x, as there is a very simple update path.

So likely this issue is no longer relevant. If you think that this issue still applies please open it again and target the new branch. All issues on the 2.x branch were closed in bulk.